src/main/kotlin/org/rust/ide/inspections/lints/RsDoubleMustUseInspection.kt

```diff
@@ -29,9 +29,9 @@ private class FixRemoveMustUseAttr : LocalQuickFix {
 class RsDoubleMustUseInspection : RsLintInspection() {
     override fun getLint(element: PsiElement): RsLint = RsLint.DoubleMustUse
 
-    override fun buildVisitor(holder: RsProblemsHolder, isOnTheFly: Boolean) = object : RsVisitor() {
-        override fun visitFunction(o: RsFunction) {
-            super.visitFunction(o)
+    override fun buildVisitor(holder: RsProblemsHolder, isOnTheFly: Boolean) = object : RsWithMacrosInspectionVisitor() {
+        override fun visitFunction2(o: RsFunction) {
+            super.visitFunction2(o)
```

Comment:
Is it necessary to call super method in `RsWithMacrosInspectionVisitor`?

Reply:
In this case seems not